internal/sql/parser/drop.go

```diff
@@ -39,7 +41,7 @@ func (p *Parser) parseDropTableStatement() (statement.DropTableStmt, error) {
 	// Parse table name
 	stmt.TableName, err = p.parseIdent()
 	if err != nil {
-		pErr := err.(*ParseError)
+		pErr := errors.Unwrap(err).(*ParseError)
```

Comment:
Could you rebase your branch please? This has already been merged

Reply:
done.